### ISSUE TITLE
support.9oo91e.qjz9zk is blocked

### ISSUE THREAD

**Issue by gothmog123** [Eloston/ungoogled-chromium#1106]
Hi I'm getting the aw snap! message on a lot of normal sites all the time (on Archlinux), then if I click learn more then I see:

support.9oo91e.qjz9zk is blockedRequests to the server have been blocked by an extension.
Try disabling your extensions.
ERR_BLOCKED_BY_CLIENT

any help appreciated

**Comment by jstkdng** [comment]
A recent update to ffmpeg brok chromium, an update is on the way.

**Comment by lahwaacz** [comment]
The patch is https://git.archlinux.org/svntogit/packages.git/tree/trunk/chromium-ffmpeg-4.3.patch?h=packages/chromium&id=d7ba6a0893c85d15faea19e9fdf5f04d18a8e449 or https://gitweb.gentoo.org/repo/gentoo.git/tree/www-client/chromium/files/chromium-84-mediaalloc.patch?id=fd3c6eeae58085ad2248e99ad67d8eba0f3a25ff, it seems to work even with (ungoogled) chromium version 83.

**Comment by Eloston** [comment]
Is this already fixed on Arch Linux?

**Comment by wchen342** [comment]
Yes, it should be [this patch](https://github.com/ungoogled-software/ungoogled-chromium-archlinux/blob/master/chromium-ffmpeg-4.3.patch).

**Comment by Eloston** [comment]
Closing then.

support pages being broken is discussed in another issue.

**Comment by scarshot** [comment]
Binary version 87.0.4280.141 on Linux.. Same error.

**Comment by geos-github** [comment]
Arch Binary Version 115.0.5790.110 still the same issue.
